testing-commandments.md

```diff
@@ -0,0 +1,22 @@
+# HolidayExtras Automated Testing Commandments
+
+## Thou shalt be consistent within projects
+If a project has a de-facto standard, it should be maintained. The process for changing a de-facto standard should take the form of a whiteboard meeting, not a pull request.
+
+## Thy tests should be obvious
+If the test behaviour isn't obvious by looking at it, it's no good.
+
+## Thy tests should be loosely coupled with thy code
+If I make a single code change and many tests fail, it's no good.
```

Comment:
"code" is a bit ambiguous I wonder if we can make this a bit clearer by saying it is ok for tests to be couple to the interface but not to the implementation.

If you are changing the thing you are testing (the interface) you are always going to have to change the tests by definition.